keel-api/src/test/kotlin/com/netflix/spinnaker/keel/api/TestAssetPlugin.kt

```diff
@@ -5,11 +5,11 @@ import com.netflix.spinnaker.keel.api.plugin.CurrentResponse
 import io.grpc.stub.StreamObserver
 
 class TestAssetPlugin : AssetPluginGrpc.AssetPluginImplBase() {
-  override fun current(request: Asset, responseObserver: StreamObserver<CurrentResponse>) {
+  override fun current(request: AssetContainer, responseObserver: StreamObserver<CurrentResponse>) {
```

Comment:
why does it need the partials in order to get the current state?

Reply:
Feels like we should really just pass the `AssetId`.

Reply:
Oh, here's why: Plugins won't know what to do with an `AssetId`. In the case of the AwsAssetPlugin, it's built entirely around the ability to inspect the request to decide what to do with it.

Reply:
That's fair.

Reply:
It doesn't need the partials, though. Right?